Dockerfile

```diff
@@ -13,6 +13,9 @@ ENV LOCALSTACK_BUILD_GIT_HASH=${LOCALSTACK_BUILD_GIT_HASH}
 ENV LD_LIBRARY_PATH=/usr/lib/jvm/java-11/lib:/usr/lib/jvm/java-11/lib/server
 ENV LOCALSTACK_HOSTNAME=localhost
 
+# Set edge bind host so localstack can be reached by other containers
+ENV EDGE_BIND_HOST=0.0.0.0
```

Comment:
Looks like some Lambda tests are failing in CI (because the Lambda containers are trying to access the LocalStack APIs via the Docker bridge network interface). I guess we also need to set `EDGE_BIND_HOST=0.0.0.0` for the tests / CI builds (e.g., in the startup logic in `tests/integration/conftest.py`).